backend/src/main/java/ai/verta/modeldb/versioning/blob/diff/ConflictGenerator.java

```diff
@@ -135,6 +135,7 @@ private static AutogenVersionEnvironmentDiff getVersionDiff(
       AutogenVersionEnvironmentDiff a,
       AutogenVersionEnvironmentDiff b,
       AutogenVersionEnvironmentBlob c) {
+    if ((a == null && b == null) || (a != null && a.equals(b))) return null;
```

Comment:
addition like these  are resolving the actual issue reported in VR-4156

Reply:
Could you expand on why they resolve?

Reply:
during the merge attempt, we collect the locations of the conflicts. A location could have a python env blob , which has  the same python version but a conflict in requirements . this code would remove the BLOBs which are not in conflict mainly by `a.equals(b)`.